### ISSUE TITLE
Create a "clone" of the Reolink app in Flutter

### ISSUE THREAD

**Issue by zachheine** [QuantumEntangledAndy/neolink#223]
**Is your feature request related to a problem? Please describe.**
No problem, just an idea. This is beyond my coding abilities otherwise I would try to make it.

**Describe the solution you'd like**
This would probably sit outside of this repo, but would love to see a "clone" of the Reolink app in Flutter.  I assume that because this is in Rust it would be possible to run it in a Flutter project such as this one with some mods: https://github.com/mitchross/multi_camera_dashboard.

**Comment by Jaywalker** [comment]
Does Tauri support mobile? I'd be mostly interested in an open source alternative to the Reolink mobile app

**Comment by QuantumEntangledAndy** [comment]
Yes I've built a few apps on iOS and android for it although you have to use the beta it works quite well. Mostly it just sets up a webview and an ipc to communicate with rust from the js
